packages/plugin-init/sources/commands/init.ts

```diff
@@ -248,7 +248,8 @@ export default class InitCommand extends BaseCommand {
         quiet: true,
       });
 
-      if (!xfs.existsSync(ppath.join(this.context.cwd, `.git`))) {
+      const isGitRepo = (await execUtils.execvp(`git`, [`rev-parse`, `--is-inside-work-tree`], {cwd: this.context.cwd})).code === 0;
```

Comment:
Probably too terse to be merged directly, but I wasn't sure which style I should pick since there's no automated formatter available. I'll apply your stylistic suggestion if you have one.

Reply:
name: Check Code  @Formatting
```suggestion
      const isGitRepo = (await execUtils.execvp(`git`, [`rev-parse`, `--is-inside-work-tree`], {cwd: this.context.cwd})).code === 0;
```

on: 
  pull_request:
    branches: [main]

jobs:
  check-formatting:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      
      - name: Setup Node
        uses: actions/setup-node@v3
        with:
          node-version: '20'
          
      - name: Install Prettier
        run: npm install -g prettier
          
      - name: Check formatting
        run: prettier --check "src/**/*.{js,jsx,ts,tsx,json,css,md}"